### ISSUE TITLE
SCVMM Domain not configured properly

### ISSUE THREAD

**Issue by apellini** [CiscoDevNet/terraform-provider-mso#114]
### Terraform Version

Terraform v1.1.5
on linux_amd64
+ provider registry.terraform.io/ciscodevnet/mso v0.4.1

### MSO version

* V 2.2(3j)

### APIC version and APIC Platform for Site Level Resources

* V 4.2(3I) and on-prem

### Affected Resource(s)

* mso_schema_site_anp_epg_domain

### Terraform Configuration Files

```hcl
resource "mso_schema_site_anp_epg_domain" "MSO_SCHEMA_SITE_ANP_EPG_DOMAIN_VMM_MS" {
  anp_name = "ANP_ONE"
  deploy_immediacy = "lazy"
  dn = "SCVMM"
  domain_type = "VMM"
  epg_name = "EPG_ONE"
  resolution_immediacy = "lazy"
  schema_id = "1234454535435"
  site_id = "1234454535435"
  template_name = "Template1"
  vlan_encap_mode = "static"
  port_encap_vlan = 3666
  port_encap_vlan_type = "vlan"
}
```

### Debug Output

No error reported on apply.
--->

### Panic Output

No panic code.

### Expected Behavior

VMM Domain configured for MS integration with correct dn.

### Actual Behavior

VMM Domain configured but on MSO is not associated to SCVMM Domain

### Steps to Reproduce

1. `terraform apply`

**Comment by apellini** [comment]
do you have any news?

**Comment by shrsr** [comment]
@apellini The issue is fixed: #130 and it will be incorporated in our next release.
